requirements/extra.txt

```diff
@@ -4,7 +4,7 @@ matplotlib>3.1
 horovod>=0.21.2  # no need to install with [pytorch] as pytorch is already installed
 omegaconf>=2.0.1
 torchtext>=0.5
-onnx>=1.7.0
+# onnx>=1.7.0
```

Comment:
@Borda Could you comment the reason why?

Reply:
it seems to be a duplicate to the following `onnxruntime`